### PR TITLE
T370367: [ALT TEXT] Add overflow menu and actions for alt text

### PR DESCRIPTION
**Pull request by l-olson1214**
**Phabricator:** 
https://phabricator.wikimedia.org/T370367
### Notes
* Added return functionality from FAQ Page
* Added ellipsesCircle image to trailing nav bar
* Does not have tooltip functionality yet
* Because of simulator / no test device, need double check on email functionality

### Test Steps
1. 

###  Screenshots/Videos

https://github.com/user-attachments/assets/c35f2e13-7d54-47b1-8ed4-4c944fe5cf85

### CODE REVIEW

File: Wikipedia/Code/ArticleViewController.swift [hunk 1]
```diff
@@ -1045,6 +1045,14 @@ private extension ArticleViewController {
         if let altTextExperimentViewModel {
             self.navigationItem.titleView = nil
             self.title = altTextExperimentViewModel.localizedStrings.articleNavigationBarTitle
+
+            let rightBarButtonItem = UIBarButtonItem(
+                image: UIImage(systemName: "ellipsis.circle"),
```
Comment (mazevedofs) [inline Wikipedia/Code/ArticleViewController.swift:1050]: We still have several image literals all over the code, but we're trying to move on to using our WMFSFSymbolIcon component for consistency.

So this bit could be:

`UIBarButtonItem(image: WMFSFSymbolIcon.for(symbol: .ellipsisBubble)`

File: Wikipedia/Code/ArticleViewController.swift [hunk 2]
```diff
@@ -1045,6 +1045,14 @@ private extension ArticleViewController {
...
+                primaryAction: nil,
+                menu: overflowMenu)
+            navigationItem.rightBarButtonItem = rightBarButtonItem
+            rightBarButtonItem.tintColor = .link
```
Comment (mazevedofs) [inline Wikipedia/Code/ArticleViewController.swift:1054]: Our theming is a little weird (we can talk more about it!), so we get colors from the theme like this:

`theme.colors.link`

This class already references the Theme class that propagates theme changes across the app.

File: Wikipedia/Code/ArticleViewController.swift [hunk 3]
```diff
@@ -1055,7 +1063,55 @@ private extension ArticleViewController {
...
+            let emailBodyLine2 = WMFLocalizedString("image-recommendations-email-second-line", value: "- [Describe specific problem]", comment: "Text for Image recommendations pre-filled issue report email. This text is intended to be replaced by the user with a description of the problem they are encountering")
+            let emailBodyLine3 = WMFLocalizedString("image-recommendations-email-third-line", value: "The behavior I would like to see is:", comment: "Text for Image recommendations pre-filled issue report email")
+            let emailBodyLine4 = WMFLocalizedString("image-recommendations-email-fourth-line", value: "- [Describe proposed solution]", comment: "Text for Image recommendations pre-filled issue report email. This text is intended to be replaced by the user with a description of a user suggested solution")
+            let emailBodyLine5 = WMFLocalizedString("image-recommendations-email-fifth-line", value: "[Screenshots or Links]", comment: "Text for Image recommendations pre-filled issue report email. This text is intended to be replaced by the user with a screenshot or link.")
```
Comment (mazevedofs) [inline Wikipedia/Code/ArticleViewController.swift:1103]: For the strings here from line 1097 forward:

If the copy is the same, you can move it into CommonStrings, as we talked about during eng sync. You can keep the keys the same so they don't need to be translated again (changing keys will kick off that whole process as it was a new string).

File: Wikipedia/Code/ArticleViewController.swift [hunk 4]
```diff
@@ -1055,7 +1063,55 @@ private extension ArticleViewController {
...
+    private var overflowMenu: UIMenu {
+        let learnMore = UIAction(title: CommonStrings.learnMoreTitle(), image: UIImage(systemName: "info.circle"), handler: { [weak self] _ in
+            self?.goToFAQ()
+        })
+        
+        let tutorial = UIAction(title: CommonStrings.tutorialTitle, image: UIImage(systemName: "lightbulb.min"), handler: { [weak self] _ in
+            self?.showTutorial()
+        })
+
+        let reportIssues = UIAction(title: CommonStrings.problemWithFeatureTitle, image: UIImage(systemName: "flag"), handler: { [weak self] _ in
```
Comment (mazevedofs) [inline Wikipedia/Code/ArticleViewController.swift:1076]: Same for the images here - they could be WMFSFSymbolIcon. You can add a new SFSymbol if we don't have it already

**Comment by mazevedofs** [comment]
Hi @l-olson1214 👋 
Great start! Nice to see you diving into our codebase 

I made some minor comments and found one issue that we can work on together if you'd like:
- After clicking "Learn more" to the in-app web view, the half-sheet modal is not presented with the alt text task when returning to the feature. That is a recurring challenge with that kind of component, so some work is expected to be done around it. 

In the PR description section, when it's ready for final review, you can add more detail about your changes, any comments or considerations (or even questions for the reviewer)
We also add test steps on test notes. 
Screenshots are not required. You can add them if it's a new UI or if it can be helpful for the reviewer.

**Review by mazevedofs — CHANGES_REQUESTED** [review]

**Review by mazevedofs — CHANGES_REQUESTED** [review]
Looking great! The send email func is working. I made a comment about adding the translations, let me know if you need help with that

File: Wikipedia/Code/ArticleViewController.swift [hunk 5]
```diff
@@ -1065,7 +1090,63 @@ private extension ArticleViewController {
...
+        let emailBodyLine3 = WMFLocalizedString("alt-text-email-third-line", value: "The behavior I would like to see is:", comment: "Text for Alt Text pre-filled issue report email")
+        let emailBodyLine4 = WMFLocalizedString("alt-text-email-fourth-line", value: "- [Describe proposed solution]", comment: "Text for Alt Text pre-filled issue report email. This text is intended to be replaced by the user with a description of a user suggested solution")
+        let emailBodyLine5 = WMFLocalizedString("alt-text-email-fifth-line", value: "[Screenshots or Links]", comment: "Text for Alt Text pre-filled issue report email. This text is intended to be replaced by the user with a screenshot or link.")
+        let emailBody = "\(emailBodyLine1)\n\n\(emailBodyLine2)\n\n\(emailBodyLine3)\n\n\(emailBodyLine4)\n\n\(emailBodyLine5)"
+        let mailto = "mailto:\(emailAddress)?subject=\(emailSubject)&body=\(emailBody)".addingPercentEncoding(withAllowedCharacters: .urlQueryAllowed)
+
```
[the remark this answers is not in the document]
Comment (l-olson1214) [reply Wikipedia/Code/ArticleViewController.swift:1142]: Added! 
Out of curiosity, when translating Chinese (or different characters in general), what tool do you use to get them to unicode?

[the remark this answers is not in the document]
Comment (mazevedofs) [reply Wikipedia/Code/ArticleViewController.swift:1142]: These files don't need Unicode characters, so you can copy and paste them directly. Did you push these changes? I can't see them on github's interface

**Review by mazevedofs — APPROVED** [review]
Looks great!

**Pull request merged**